### PR TITLE
Dummy PR: Move long-running fork upstream

### PR DESCRIPTION
**Pull request by ghost**
##### SUMMARY
This PR should never be merged.  Separate PRs are being prepared based on the diffs in this PR.

##### ISSUE TYPE
- Bugfix
- New Features

##### COMPONENT NAME
- core infrastructure plays/roles
- ocp4-workshop config
- new ansible-skylight config

##### ADDITIONAL INFORMATION
DO NOT merge this PR - it will be deleted when no longer needed.

### CODE REVIEW

File: ansible/cloud_providers/ec2_infrastructure_deployment.yml [hunk 1]
```diff
@@ -62,6 +62,17 @@
       import_role:
         name: infra-common-ssh-config-generate
 
+    - name: Stop SSH/Socks proxy for Windows proxying through bastion
```
Comment (joelbirchler) [inline ansible/cloud_providers/ec2_infrastructure_deployment.yml:65]: 1023

File: ansible/configs/ansible-skylight/README.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
+= ansible-skylight config
```
Comment (joelbirchler) [inline ansible/configs/ansible-skylight/README.adoc:1]: 1024

File: ansible/configs/ansible-skylight/destroy_env.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (joelbirchler) [inline ansible/configs/ansible-skylight/destroy_env.yml:1]: 1024

File: ansible/configs/ansible-skylight/env_vars.yml [hunk 4]
```diff
@@ -0,0 +1,398 @@
+---
```
Comment (joelbirchler) [inline ansible/configs/ansible-skylight/env_vars.yml:1]: 1024

File: ansible/configs/ansible-skylight/files/cloud_providers/ec2_cloud_template.j2 [hunk 5]
```diff
@@ -0,0 +1,325 @@
+AWSTemplateFormatVersion: "2010-09-09"
```
Comment (joelbirchler) [inline ansible/configs/ansible-skylight/files/cloud_providers/ec2_cloud_template.j2:1]: 1024

File: ansible/configs/ansible-skylight/files/hosts_template.j2 [hunk 6]
```diff
@@ -0,0 +1,44 @@
+[windows_hosts]
```
Comment (joelbirchler) [inline ansible/configs/ansible-skylight/files/hosts_template.j2:1]: 1024

File: ansible/configs/ansible-skylight/files/repos_template.j2 [hunk 7]
```diff
@@ -0,0 +1,35 @@
+[rhel-7-server-rpms]
```
Comment (joelbirchler) [inline ansible/configs/ansible-skylight/files/repos_template.j2:1]: 1024

File: ansible/configs/ansible-skylight/files/status.j2 [hunk 8]
```diff
@@ -0,0 +1,5 @@
+{{ "%-60s" | format('Instance') }} State      Type
```
Comment (joelbirchler) [inline ansible/configs/ansible-skylight/files/status.j2:1]: 1024

File: ansible/roles/infra-ec2-create-inventory/defaults/main.yml [hunk 9]
```diff
@@ -0,0 +1,2 @@
+---
+win_connect_method: winrm
```
Comment (joelbirchler) [inline ansible/roles/infra-ec2-create-inventory/defaults/main.yml:2]: 1023?

File: ansible/roles/status-report-install/tasks/main.yml [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+- name: Create application folder
+  file:
+    name: "/opt/status/src"
+    state: directory
+- name: Retrieve application source files
+  git:
+    clone: yes
+    force: yes
+    dest: "/tmp/status"
+    repo: "http://github.com/jimrigsbee/api.git"
```
Comment (rmahroua) [inline ansible/roles/status-report-install/tasks/main.yml:12]: @joelbirchler @jimrigsbee

**Pull request closed without merging**